src/PAK-Samples/MapReferenceToPaper/who-covid-19-makes-config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "InterpretBaseUrl": "http://{instance_name}.westuscloudapp.azure.com/interpret?",
+  "EvaluateBaseUrl": "http://{instance_name}.westus.cloudapp.azure.com/evaluate?",
```

Comment:
>cloudapp [](start = 52, length = 8)

do MAKES VM urls use cloudapp.azure.com?

Reply:
Yes

---
In reply to: [398907751](https://github.com/microsoft/mag-covid19-research-examples/pull/13#discussion_r398907751) [](ancestors = 398907751)